rebasehelper/exceptions.py

```diff
@@ -26,7 +26,21 @@ class RebaseHelperError(Exception):
     Class representing Error raised inside rebase-helper after intentionally
     catching some expected and well known exception/error.
     """
-    pass
+
+    def __init__(self, *args, **kwargs):
+        """
+        Init method for RebaseHelperError exception
+
+        :param args: error message
+        :param kwargs: more optional arguments
```

Comment:
The docstring is not very accurate. `Init method for RebaseHelperError exception` - `Constructor of RebaseHelperError` sounds better to me. I wouldn't describe `args` and `kwargs` at all perhaps, or if so, I'd use `error message plus optional formatting arguments` for `args`, and `keyword arguments` for `kwargs`, with added description of `logfile`.